client/lib/plans/features-list.tsx

```diff
@@ -1958,8 +1958,8 @@ export const FEATURES_LIST: FeatureList = {
 	},
 	[ FEATURE_PREMIUM_THEMES_V2 ]: {
 		getSlug: () => FEATURE_PREMIUM_THEMES_V2,
-		getTitle: () => i18n.translate( 'Premium themes' ),
-		getIcon: () => <img src={ Theme2Image } alt={ i18n.translate( 'Premium themes' ) } />,
+		getTitle: () => i18n.translate( 'Explorer themes' ),
```

Comment:
This change will affect the the plans grid. 

I haven't changed the title/description as it's using premium in a more descriptive sense and is used to supplement the "Explorer themes" label, e.g. in alt text.

Reply:
If this affects the Plans Grid, I think that we should add the `String Freeze` label to this PR. PaYJgx-3sX-p2

Reply:
yeah I was anticipating some feedback on the strings so I was holding off, and lo and behold!

Reply:
When we merge these changes the Plans Grid will show the `Creator Themes` feature regardless of if we shipped the new tier or not.